modules/oadp-configuring-client-burst-qps.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-configuring-client-burst-qps_{context}"]
+= Configuring the DPA with client burst and QPS settings
+
+The queries per second (QPS) setting determines the number of client requests or queries that are sent to the `velero` server. The burst setting determines how many requests from the client can be stored for processing.
+
+You can set the burst and QPS values of the `velero` server by configuring the Data Protection Application (DPA) with the burst and QPS values. You can use the `dpa.configuration.velero.client-burst` and `dpa.configuration.velero.client-qps` fields of the DPA to set the burst and QPS values.
+
+
+.Prerequisites
+
+* You have installed the {oadp-short} Operator.
+
+.Procedure
+
+* Configure the `client-burst` and the `client-qps` fields in the DPA as shown in the following example:
++
+.Example Data Protection Application
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: test-dpa
+  namespace: openshift-adp
+spec:
+  backupLocations:
+    - name: default
+      velero:
+        config:          
+          insecureSkipTLSVerify: "true"
+          profile: "default"
+          region: <bucket_region> 
+          s3ForcePathStyle: "true"
+          s3Url: <bucket_url>
+        credential:
+          key: cloud
+          name: cloud-credentials
+        default: true
+        objectStorage:
+          bucket: <bucket_name> 
+          prefix: velero
+        provider: aws
+  configuration:
+    nodeAgent:
+      enable: true
+      uploaderType: restic
+    velero:
+      client-burst: 300 # <1>
+      client-qps: 500 # <2>
```

Comment:
I would update the example to have these either the same or have burst larger than qps. Maybe just reverse the two numbers here.

Reply:
Done. thanks.